index.html

```diff
@@ -0,0 +1,312 @@
+          </div>
+          <div class="col-md-2 px-0">
+            <div class="card schedule-card">
+              <img
+                src="./assets/exhibition.png"
+                class="card-img-top"
+                alt="icon"
+              />
+              <div class="card-body d-flex flex-column align-items-center">
+                <h5 class="card-title">Memory Lane</h5>
+                <p class="card-text">
+                  Tour the preserved colonial farmhouse, a kitchen in a separate
+                  building, a coffee-drying plant in the woodland, and an
+                  agricultural college that showcases plants.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-2 px-0">
+            <div class="card schedule-card">
+              <img src="./assets/forum.png" class="card-img-top" alt="icon" />
+              <div class="card-body d-flex flex-column align-items-center">
+                <h5 class="card-title">Story time</h5>
+                <p class="card-text">
+                  Explore the historical transformation and hear about
+                  interesting tales of our ancestry and explore the different
+                  ways in which our predecessors lived
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-2 px-0">
+            <div class="card schedule-card">
+              <img
+                src="./assets/workshop.png"
+                class="card-img-top"
+                alt="icon"
+              />
+              <div class="card-body d-flex flex-column align-items-center">
+                <h5 class="card-title">Culture</h5>
+                <p class="card-text">
+                  Museums display much more than artifacts; Museum Culture makes
+                  us on a tour through the complex of ideas, values and symbols
+                  in society that need to be represented.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-2 px-0">
+            <div class="card schedule-card">
+              <img src="./assets/ignite.png" class="card-img-top" alt="icon" />
+              <div class="card-body d-flex flex-column align-items-center">
+                <h5 class="card-title">Socialize</h5>
+                <p class="card-text">
+                  The Mamboleo Historia Global Summit brings together more than
+                  forty international speakers to share action orientated
+                  presentations on culture.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="program-schedule">SEE THE WHOLE PROGRAM</p>
+        <button class="btn-program">Join Mamboleo Summit 2022</button>
+      </section>
+      <section class="guests text-center">
+        <div class="container">
+          <h2 class="guests-title">Featured speakers</h2>
```

Comment:
- [ ] Kindly add some margins to the featured speakers so that it won't be too close to the other sections as it is on the design template

|  Design Template   | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181393011-1af7c080-ec37-42ce-a004-56e12c3a953a.png)  | ![image](https://user-images.githubusercontent.com/63473584/181393076-376dd617-ad0d-4aca-badc-479f2a3bdc6a.png)  |